### PR TITLE
Viite 700 editing continuity and road type for project link

### PR DESCRIPTION
**Pull request by plourenco90**

### CODE REVIEW

**Comment by plourenco90** [comment]
Can´t be merged before the VIITE-779

File: digiroad2-oracle/src/main/resources/db/migration/V131__add_required_attributes.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE PROJECT_LINK ADD ELY NUMBER;
```
Comment (tvenhola) [inline digiroad2-oracle/src/main/resources/db/migration/V131__add_required_attributes.sql]: Can't replace a migration file that is already in master

Comment (plourenco90) [reply digiroad2-oracle/src/main/resources/db/migration/V131__add_required_attributes.sql]: Ok, i will change it for V132

**Comment by tvenhola** [comment]
Merging master into this branch

**Pull request merged**